src/components/Wizard/CreateVmWizard/BasicSettingsTab.js

```diff
@@ -4,6 +4,7 @@ import { get } from 'lodash';
 import { FormFactory } from '../../Form/FormFactory';
 import { getName } from '../../../utils/selectors';
 import { getTemplate } from '../../../utils/templates';
+import { validateDNS1123SubdomainValue, getValidationObject } from '../../../utils/validations';
```

Comment:
do we know if there is any similar validation on the api side? just wondering if we are creating a mix UX where some validations are enforced only in the UI.